index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
this code is clean, with consistent indentation and spacing throughout! well done!